plutus-conformance/README.md

```diff
@@ -39,7 +39,7 @@ If evaluation fails with an error, the expected output will show "evaluation err
 
 3. An untyped plutus core program
 
-This means the input file successfully evaluates to the output program as per the specification. The evaluated program is represented in the concrete syntax.
+This means the input file successfully evaluates to the output program as per [the specification](https://github.com/input-output-hk/plutus/blob/master/plutus-core-spec/draft-new-specification/plutus-core-specification.pdf). The evaluated program is represented in the concrete syntax.
```

Comment:
@kwxm any reason the spec is in a `draft-new-specification` dir? I'm just worried that the spec will be moved and the link will be broken. Could we perhaps put it in `plutus-core-spec` directly? Thanks.

Reply:
> @kwxm any reason the spec is in a `draft-new-specification` dir?

It's been sitting about for a long time and there are still some gaps in it (notably it says nothing about TPLC).  I think it also needs to be reviewed by experts.  We are trying to get moving on upgrading it to be the official specification though: we've got issues in JIRA for that.  I hope to make some progress on this quite soon.

Reply:
OK thanks for the update. Not trying to rush you! I'll remove the link for now then.